mcode/mcode-extraction-framework
# Adding event.coding.version and event.text to CTCAE Extractor

# Summary
Adds an optional version field for the event coding and a required text field for the event of the CSVCTCAdverseEventExtractor
## New behavior
The CSV template for the CSVCTCAdverseEventExtractor now has a new required field of `adverseEventText` and a new optional field `adverseEventCodeVersion`
## Code changes
- `CTCAdverseEventTemplate.js` modified to add the new fields
- `CSVCTCAdverseEventExtractor.js` modified to have the new fields in the CSV template and check for new required field
- All example CSVs and `CSV_Templates.xlsx` changed to have new fields
- Relevant tests and test fixtures now test for the new fields
# Testing guidance
- Run the client with the CSVCTCAdverseEventExtractor and see that the event now has a coding version and text field
- Make sure the new fields work as described in the profile: https://build.fhir.org/ig/standardhealth/fsh-ae/StructureDefinition-ctc-adverse-event.html
- Ensure all tests still pass and that the test for the CTCAE properly test the new fields
